src/Chapter1.hs

```diff
@@ -620,7 +633,12 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Integral a => a -> a
+sumLast2 n = secondLastDig + lastDig
+  where
+    lastDig = mod an 10
+    secondLastDig = mod (div an 10) 10
+    an = abs n
```

Comment:
On a side note, there is a standard function as well that combines both `div` and `mod` at once: `divMod`.

It returns the pair, so you can pattern match on that and get both variabless at once:

```haskell
let (a, b) = abs n `divMod` 10
```



That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter: the standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that is actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(first, second) = divMod (mod x 100) 10
```

You can see how we could pattern match on the pair 🙂

Reply:
That's cool! I have changed it to use `divMod`. Thanks!